mephisto/core/task_launcher.py

```diff
@@ -32,8 +32,8 @@
 
 logger = get_logger(name=__name__, verbose=True, level="debug")
 
-UNIT_GENERATOR_WAIT_SECONDS = 10
-ASSIGNMENT_GENERATOR_WAIT_SECONDS = 0.5
+UNIT_GENERATOR_WAIT_SECONDS = 1
+ASSIGNMENT_GENERATOR_WAIT_SECONDS = 0.05
```

Comment:
Why change these times?

Reply:
there was a test in the `test_operator.py` making sure that consecutive assignments finish after certain amount of time, when they are as large as before it fails, so I changed it and was about to ask you (how can we define these numbers efficiently? and should the test in the `test_operator.py` be changed later?)

Reply:
I'd update the `test_operator` to use a timeout style test rather than a direct check, like the style of:
```
start_time = time.time()
while (condition is not met):
  assert time.time - start_time is not more than some amount
```
as the timeouts defined initially here seem better than these really rapid ones.